ios/ble/peripheral/PeripheralCommunicatorDelegate.swift

```diff
@@ -11,4 +11,6 @@ protocol PeripheralCommunicatorProtocol: AnyObject {
 protocol WalletProtocol: AnyObject {
     func onIdentifyWriteSuccess()
     func onDisconnectStatusChange(data: Data?)
+    func createConnectionHandler()
```

Comment:
On what basis are we putting a method in either WalletProtocol or PeripheralCommunicatorProtocol?

For example:
`createConnectionHandler` is being called from PeripheralDelegate while `setVeriferKeyOnSameIdentifier` is called from `CentralManagerDelegate`

Reply:
WalletProtocol and PeripheralCommunicatorProtocol methods are created to communicate with Wallet and Transferhandler(peripheral stuff) from Central respectively.